swhkd/src/daemon.rs

```diff
@@ -195,8 +195,8 @@ async fn main() -> Result<(), Box<dyn Error>> {
             let log = log.clone();
 
             // Set the user and group id to the invoking user for the thread
-            setgid(Gid::from_raw(invoking_uid)).unwrap();
-            setuid(Uid::from_raw(invoking_uid)).unwrap();
+            setuid(Uid::from_raw(invoking_uid)).expect(&format!("Failed to set group-id to {}", invoking_uid));
```

Comment:
The call to gid is first then it's uid, it seems it got swapped here. It seems like a typo (going by the error log).